packages/react/presence/src/Presence.tsx

```diff
@@ -47,6 +47,11 @@ function usePresence(present: boolean) {
     },
   });
 
+  React.useEffect(() => {
+    const currentAnimationName = getAnimationName(stylesRef.current);
+    prevAnimationNameRef.current = state === 'mounted' ? currentAnimationName : 'none';
```

Comment:
The cause of the original issue was that when animating out the first time the `prevAnimationNameRef.current` was becoming `hGfvx` or something (the name of the close animation) and then when remounting there was no animation so `prevAnimationNameRef.current` was still set to the close animation.

That meant that the second time it tried to close, the previous and current animation values would be the same and it wouldn't suspend unmount as it though there wasn't an animation initiating.

This change ensures that when the component unmounts we always set the previous animation to `none`.